docs/package-migration-process-guide.md

```diff
@@ -0,0 +1,201 @@
+# Package Migration Process Guide
+
+This document outlines the process for migrating a MetaMask library into the core monorepo. The migration target is assumed to comply with the requirements defined by [`metamask-module-template`](https://github.com/MetaMask/metamask-module-template) and [`module-lint`](https://github.com/MetaMask/module-lint).
+
+## Phase A: **Preparation** in the _Source Repo_
+
+### **[PR#1]** 1. Add the following migration notice to the README
+
+```markdown
+<table><tr><td><p align="center"><b>⚠️ PLEASE READ ⚠️</b></p><p align="center">This package is currently being migrated to our <a href="https://github.com/MetaMask/core"><code>core</code></a> monorepo. Please do not make any commits to this repository while this migration is taking place, as they will not be transferred over. Also, please re-open PRs that are under active development in the core repo.</p></td></tr></table>
+```
+
+- [Example PR](https://github.com/MetaMask/eth-json-rpc-provider/pull/38)
+
+### 2. Disable `dependabot` dependency upgrades
```

Comment:
What do you think about explaining how to do this?

Reply:
Fixed here 2cf03afa525a73e9a0e9636e27a8167eae6b8b87